x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/deprecation/deprecation-indexing-ilm-policy.json

```diff
@@ -4,9 +4,18 @@
       "actions": {
         "rollover": {
           "max_primary_shard_size": "10gb",
-          "max_age": "14d"
+          "max_age": "30d"
         }
       }
     }
+  },
+  "delete": {
+    "min_age": "30d",
+    "actions":{
+      "delete": {}
+    }
+  },
+  "_meta": {
+    "description": "ILM policy used for deprecation logs"
```

Comment:
Do we also want "managed": true for this policy?

Reply:
I think we want users to be able to easily modify this one, but open to counter opinion.

Reply:
on second thought ... maybe it should be managed we don't want to encourage deletion and is associate with a hidden data stream. (and the name starts with a `.`) 
thoughts ?

Reply:
+1 to have as managed, it's more of a suggestion anyway :)